components/03-modules/slider/slider.js

```diff
@@ -0,0 +1,18 @@
+/* globals lory */
+
+(function() { // eslint-disable-line
+  'use strict'
```

Comment:
Missing semicolon.
I see you are using it later in code, keep one convention.

Reply:
never ever before put `;` here and in the rest of code yes, don't know is it was wrong or right